job/gatherActiveLdus.js

```diff
@@ -21,7 +21,7 @@ const getBookingIds = async () => {
   }
 
   const { rows } = await db.query(query)
-  return rows.map(row => row.booking_id)
+  return rows.map((row) => row.booking_id)
```

Comment:
latest prettier adds brackets to single args which accounts for 700 changes across the files

Reply:
Is that an improvement? Can it be turned off?  Perhaps its best to accept that change now that you've implemented it!